docs/docs-components/LocalizationSection.js

```diff
@@ -0,0 +1,73 @@
+// @flow strict
+import { type Node } from 'react';
+import { SlimBanner } from 'gestalt';
+import MainSection from './MainSection.js';
+import SandpackExample from './SandpackExample.js';
+
+type Props = {|
+  notes?: string,
+  name: string,
+  code?: () => Node,
+  layout?: 'row' | 'column',
+  previewHeight?: number,
+  children?: Node,
+  noDefaultLabelProvider?: boolean,
+  noBaseText?: boolean,
+|};
+
+export default function AccessibilitySection({
```

Comment:
Is this intentional or should this be `LocalizationSection`?

Reply:
Good catch. i'll fix in follow up PR!